### PR TITLE
Add OWNERS and pr template for mungegithub

### PR DESCRIPTION
**Pull request by yutongz**
Add required files for Mungegithub.

OWNERS file defines who can approve prs in this repo and who are recommended reviewer.
PR template helps people add release-note when creating prs. Simply add release-note or "NONE" between ```, then mungegithub will add proper release-note label. More details at https://github.com/istio/istio/pull/525

More details about [Mungegithub](https://github.com/istio/test-infra/tree/master/mungegithub/deployment)

[Related pr](https://github.com/istio/mixer/pull/1024)

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+reviewers:
```
Comment (myidpt) [inline OWNERS]: Also add myidpt and wenchenglu as reviewers/approvers?

Comment (yutongz) [inline OWNERS]: Absolutely!

**Comment by yutongz** [comment]
PTAL

**Comment by yutongz** [comment]
@lookuptable  I don't have access to merge this, could you do it for me, thanks!

**Pull request merged**